lib/logger/lib/logger/utils.ex

```diff
@@ -74,6 +74,12 @@ defmodule Logger.Utils do
     truncate_n_list(list, n, [])
   end
 
+  defp truncate_n(other, _n) do
+    raise ArgumentError,
```

Comment:
I think this is enough. An alternative could be to call `List.to_string/1` on the whole list here in order to get an even nicer error, but I think this is fine.

Reply:
This is fine by me, yeah.